go-controller/pkg/ovn/controller/services/services_controller.go

```diff
@@ -355,19 +370,24 @@ func (c *Controller) syncService(key string) error {
 	}
 
 	// Build the abstract LB configs for this service
-	perNodeConfigs, clusterConfigs := buildServiceLBConfigs(service, endpointSlices)
+	perNodeConfigs, templateConfigs, clusterConfigs := buildServiceLBConfigs(service, endpointSlices,
+		c.useLBGroups, c.useTemplates)
 	klog.V(5).Infof("Built service %s LB cluster-wide configs %#v", key, clusterConfigs)
 	klog.V(5).Infof("Built service %s LB per-node configs %#v", key, perNodeConfigs)
+	klog.V(5).Infof("Built service %s LB template configs %#v", key, templateConfigs)
 
 	// Convert the LB configs in to load-balancer objects
 	nodeInfos := c.nodeTracker.allNodes()
 	clusterLBs := buildClusterLBs(service, clusterConfigs, nodeInfos, c.useLBGroups)
+	templateLBs := buildTemplateLBs(service, templateConfigs, nodeInfos,
+		c.nodeTracker.nodeIPV4Template, c.nodeTracker.nodeIPV6Template)
```

Comment:
Not the best argument but I mimicked what was done for LBs.  They also need nodeInfos.  Like you said, there should be eventual consistency.  However, if that's wrong we need to synchronize all LB processing, this doesn't apply only to template LBs.

Reply:
I see now that you probably were referring to `c.nodeTracker.nodeIPV4Template, c.nodeTracker.nodeIPV6Template`.  I'll fix it so that we don't access them directly.